### ISSUE TITLE
Can't login wordpress.com; can't import settings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#359]
```
What's the problem?

i can't login (can't configure scribefire) my blog account on wordpress.com 
since i started using Firefox 4.0.

it happens the same in every version since 4.0, aurora, nightly etc

i still can work with Firefox 3.6.

i exported my config from Firefox 3.6 but i was unable to import it with 
Firefox 4.x and above.

What browser are you using?

Firefox 4.x and above



What version of ScribeFire are you running?

1.7
```

-----
Original issue reported on code.google.com by `rikhard....@gmail.com` on 10 Jun 2011 at 1:24

**Comment by GoogleCodeExporter** [comment]
```
Try again with ScribeFire 4; the wordpress.com code was updated there.
```

-----
Original comment by `cfinke` on 3 Nov 2011 at 3:24
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with version 4.3.3
```

-----
Original comment by `scribefi...@gmail.com` on 18 Jun 2014 at 8:27
* Changed state: **Fixed**
